whylog/client/searchers.py

```diff
@@ -40,6 +44,7 @@ def _reverse_from_offset(self, offset, buf_size=BufsizeConsts.STANDARD_BUF_SIZE)
             fh.seek(offset)
```

Comment:
I agree that we should avoid using such shortcuts, but I'am not sure if `file_path` is a proper name for this variable - due to its reference to path. Maybe `opened_file` would be better?